src/vcpu.rs

```diff
@@ -87,8 +90,8 @@ impl axvcpu::AxArchVCpu for Aarch64VCpu {
 
     fn run(&mut self) -> AxResult<AxVCpuExitReason> {
         self.restore_vm_system_regs();
-        self.run_guest();
-        self.vmexit_handler()
+        let exit_reason: usize = self.run_guest();
```

Comment:
Why is it necessary to add a type annotation here?

Reply:
emm，`mov x0, #1` operation width equal to usize.